packages/unison/unison.2.51.3/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "juergen@hoetzel.info"
+authors: [
+  "Jürgen Hötzel <juergen@hoetzel.info>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "mailto:unison-users@yahoogroups.com "
+dev-repo: "git://github.com/bcpierce00/unison.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
+  "dune" {>= "1.3"}
+  "lablgtk"
```

Comment:
I am not sure if the other failures depend on this, but from the various errors I think we may need some bounds for lablgtk as well